config.template.py

```diff
@@ -60,6 +60,16 @@ class Config:
     RUNJOB_TIMEOUT = 60
     COPYOUT_TIMEOUT = 30
 
+    # Docker constants
+    BOOT2DOCKER_INIT_TIMEOUT = 5
+    BOOT2DOCKER_START_TIMEOUT = 30
+    BOOT2DOCKER_ENV_TIMEOUT = 5
+    DOCKER_IMAGE_BUILD_TIMEOUT = 300
+    DOCKER_RM_TIMEOUT = 5
+    # Must be absolute path with trailing slash
+    # Default value of '*'' points this path to /path/to/Tango/volumes/
```

Comment:
this currently points out to " /path/to/Tango/volumes//" (two (( at the end) which causes an error